migrations/20170720002747_add_translations_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddTranslationsTable extends AbstractMigration
+{
+    /**
+     * Change method
+     */
+    public function change()
+    {
+        $this->table('translations')
+            ->addColumn('resource', 'string', ['limit' => 50, 'null' => false])
+            ->addColumn('resource_id', 'integer', ['null' => false])
+            ->addColumn('property', 'string', ['limit' => 50, 'null' => false])
+            ->addColumn('source', 'string')
```

Comment:
At some point we're going to want to mine translations, what is source in reference to?